test/compliance/generate_fbank_data.py

```diff
@@ -0,0 +1,174 @@
+import os
+import random
+import subprocess
+import torch
+import torchaudio
+
+# Path to the compute-fbank-feats executable.
+EXE_PATH = '/scratch/jamarshon/kaldi/src/featbin/compute-fbank-feats'
+
+# Path to the scp file. An example of its contents would be "my_id /scratch/jamarshon/audio/test/assets/kaldi_file.wav"
+# where the space separates an id from a wav file.
+SCP_PATH = 'scp:/scratch/jamarshon/downloads/a.scp'
+# The directory to which the stft features will be written to.
+OUTPUT_DIR = 'ark:/scratch/jamarshon/audio/test/assets/kaldi/'
+
+# The number of samples inside the input wave file read from `SCP_PATH`
+WAV_LEN = 20
+
+# How many output files should be generated.
+NUMBER_OF_OUTPUTS = 100
+
+VERBOSE = False
+
+
+def generate_rand_boolean():
```

Comment:
Since the function has a single line, you may consider to leave them onsite.

Reply:
randbool would be more consistent.